src/run_spineopt.jl

```diff
@@ -324,6 +324,31 @@ function optimize_model!(m::Model; log_level=3, calculate_duals=false, mip_solve
     # NOTE: The above results in a lot of Warning: Variable connection_flow[...] is mentioned in BOUNDS,
     # but is not mentioned in the COLUMNS section. We are ignoring it.
     @timelog log_level 0 "Optimizing model $(m.ext[:instance])..." optimize!(m)
+    if termination_status(m) == MOI.INFEASIBLE
+        compute_conflict!(m)
```

Comment:
This is nice. Would you consider putting this code in its own function, and call it from here?

Also, just to be sure this is not related to the current MR? If not, would you consider committing it to master directly?

Reply:
Yes, sorry this one was supposed to come in a separate MR. I'll try to remove it from this one and merge it separately. But it's indeed really cool, already saved me a bunch of time.